.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDateTimePeriodParserConfiguration.cs

```diff
@@ -429,6 +431,12 @@ private DateTimeResolutionResult ParseSpecificNight(string text, DateObject refe
                 beginHour = 8;
                 endHour = Constants.HalfDayHourCount;
             }
+            else if (NORegex.IsMatch(trimmedText))
+            {
+                timeStr = "TNO";

```

Comment:
There is no such concept as TNO in timex expressions. If this is the same as regular "noon" in English, then it should just be a time.
You can always test similar cases in English to see how it handles things for a ballpark idea of expected outputs.